apps/api/src/app/admin/admin.service.ts

```diff
@@ -347,16 +354,12 @@ export class AdminService {
         symbol,
         symbolMapping,
         ...(dataSource === 'MANUAL'
-          ? { name }
+          ? { name, assetClass, assetSubClass, url }
```

Comment:
`{ assetClass, assetSubClass, name, url }`

Reply:
Done (as a side note is this something prettier can be configured to enforce?)